CONTRIBUTING.md

```diff
@@ -1,64 +1,58 @@
-## Managing vendor/
```

Comment:
Hmm, I don't think we should be removing the contents of the original file, which I found particularly helpful.

Maybe add the DCO Sign off as another section to this file?

Reply:
Just so we're all clear: we actually shouldn't remove the contributing file, as we need it for scripts.